src/AppBundle/Controller/ActivityController.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace AppBundle\Controller;
+
+use AppBundle\Entity\Activity;
+use AppBundle\Form\ActivityType;
+use Doctrine\DBAL\Exception\ForeignKeyConstraintViolationException;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Component\HttpFoundation\Request;
+
+class ActivityController extends Controller
+{
+    /**
+     * @Route("/activities/view", name="activities_view")
+     */
+    public function viewAction(Request $request)
+    {
+        $activities = $this->get('app.activity')->getActivityList();
```

Comment:
tokius visada labai neblogai kelti i paprastas private funkcijas ir jas uzanotuoti. tokiu atveju pradeda veikti php stormo autocompleatas, find usages ir pan. pagalbinis funkcionalumas
/**
*@returns SomeActivityService
*/
private function getAppActivity()
{
    return $this->get('app.activity');
}